test/TestCliRolePaths.py

```diff
@@ -20,16 +20,14 @@ def run_ansible_lint(self, cwd, bin, role_path, env=None):
             env=env,
         ).communicate()
 
-        self.assertFalse(err, 'Expected no error but was ' + str(err))
-
-        return result
+        return result, stderr
```

Comment:
I don't see any point in returning stderr, you're not using it
```suggestion
        return result
```